Lib/importlib/util.py

```diff
@@ -270,3 +270,10 @@ def exec_module(self, module):
         loader_state['is_loading'] = False
         module.__spec__.loader_state = loader_state
         module.__class__ = _LazyModule
+
+
+__all__ = ['Loader', 'module_from_spec', '_resolve_name', 'spec_from_loader',
+           '_find_spec', 'MAGIC_NUMBER', '_RAW_MAGIC_NUMBER',
```

Comment:
`_find_spec` and `_RAW_MAGIC_NUMBER` look rather like private names.

Reply:
Removed with _resolve_name() as well.